src/timer.rs

```diff
@@ -411,6 +501,26 @@ macro_rules! split {
     };
 }
 
+macro_rules! split_cc {
```

Comment:
This macro can also be joined with previous as it is implemented for same timers.